testing/test_mud.sh

```diff
@@ -22,7 +22,8 @@ cat inst/run-9a02571e8f00/nodes/nmap01/activate.log
 
 echo %%%%%%%%%%%%%%%%%%%%%% Mud profile tests | tee -a $TEST_RESULTS
 rm -f local/system.yaml
-echo 'include=../config/system/muddy.conf' > local/system.conf
+echo 'include=../config/system/muddy.conf
```

Comment:
for multiline use <<EOF syntax, see daq/testing/test_many.sh for example

Reply:
Updated. PTAL..